src/js/common/components/CardForListBody.jsx

```diff
@@ -62,7 +62,7 @@ function CardForListBody (props) {
     politicalPartySvgNameWithPath = '../../img/global/svg-icons/political-party-working-families.svg';
   }
   const politicianDetailsURL = `${webAppConfig.WE_VOTE_URL_PROTOCOL + webAppConfig.WE_VOTE_HOSTNAME}${politicianBasePath}`;
-  const destinationPage = lookupPageNameAndPageTypeDict(politicianBasePath);
+  const destinationPage = lookupPageNameAndPageTypeDict(politicianDetailsURL);
```

Comment:
lookupPageNameAndPageTypeDict doesn't take a URL, it takes a path ... so for example instead of https://wevote.us/ballot it just takes /ballot.

If you look into the lookupPageNameAndPageTypeDict function, all of those `path.startsWith` would fail if the path was a URL and started with https -- they assume paths.

So the code was correct before this proposed change.